platform/viewer/src/routes/Mode/Mode.jsx

```diff
@@ -83,6 +85,8 @@ export default function ModeRoute({
     route,
     servicesManager,
     extensionManager,
+    hotkeysManager,
+    isHotkeysLoaded
```

Comment:
Is anything keying off of this? Shouldn't modes just expect hotkeys to be loaded?

Reply:
Nope, removing hook, thanks